### DISCUSSION TITLE
How do you use FizzNodes Image Select Schedule?

### DISCUSSION THREAD

**Discussion by Vektor369** [comfyanonymous/ComfyUI#6021] [Q&A]
I'm trying to find a way to give LTXVideo a few keyframes over time to interpolate between, but I can't for the life of me figure out how to do it. I noticed FizzNodes ([https://github.com/FizzleDorf/ComfyUI_FizzNodes](https://github.com/FizzleDorf/ComfyUI_FizzNodes)) has an Image Select Schedule node, and thought it might work like their Prompt Schedule, but with images instead of prompts, but I can't figure out how to use it, and I can't find any examples of it being used anywhere.

Am I going about this right? Or is there a better way? Any help would be greatly appreciated. Thanks in advance.

**Comment by LukeG89** [comment]
Check this custom node: [ComfyUI-LTXTricks](https://github.com/logtd/ComfyUI-LTXTricks)

It has an example workflow for the interpolation of keyframes: [interpolation-and-frame-setting](https://github.com/logtd/ComfyUI-LTXTricks?tab=readme-ov-file#interpolation-and-frame-setting)
But I think with LTXV you can only interpolate 2 frames at a time, check this issue: https://github.com/logtd/ComfyUI-LTXTricks/issues/7
